internal/version/version.go

```diff
@@ -18,6 +18,7 @@ package version
 
 var (
 	//TODO: Hardcode it to version string to support importing and usage in operator-sdk
-	Version   = "unknown"
-	GitCommit = "unknown"
+	Version    = "unknown"
+	GitCommit  = "unknown"
+	GitVersion = "unknown"
```

Comment:
Is this set anywhere?

Reply:
It's not. I'm actually looking at improving the version handling now. I think we should leave this PR open until we have better version number handling in place that accounts for both the main `helm-operator` binary and projects that import this repo and need to use the plugin to scaffold Makefile and Dockerfile with appropriate version numbers.

I think [`debug.ReadBuildInfo()`](https://pkg.go.dev/runtime/debug#ReadBuildInfo) may be useful to us.